myuw/dao/term.py

```diff
@@ -391,6 +403,10 @@ def add_term_data_to_context(request, context):
 
     context['first_day'] = cur_term.first_day_quarter
     context['last_day'] = cur_term.last_day_instruction
+    context["first_day_quarter"] = cur_term.first_day_quarter
+    context["last_day_instruction"] = cur_term.last_day_instruction
```

Comment:
So a week is always starts on a Sunday and ends on a Saturday.  And if there is no final, the schedule will end on the last Saturday before final week.